third_party/blink/renderer/core/messaging/message_port.cc

```diff
@@ -314,8 +314,6 @@ void MessagePort::Trace(Visitor* visitor) const {
 bool MessagePort::Accept(mojo::Message* mojo_message) {
   TRACE_EVENT0("blink", "MessagePort::Accept");
 
-  recordreplay::Assert("[RUN-1126] MessagePort::Accept");
```

Comment:
Why is this being removed?

Reply:
It did not hit. But Ok, I'll put it back in. I think I understand now.

Reply:
I added it back in